Library/Homebrew/brew.sh

```diff
@@ -195,19 +195,30 @@ then
   HOMEBREW_BASH_COMMAND="$HOMEBREW_LIBRARY/Homebrew/dev-cmd/$HOMEBREW_COMMAND.sh"
 fi
 
-if [[ "$(id -u)" = "0" && "$(/usr/bin/stat -f%u "$HOMEBREW_BREW_FILE")" != "0" ]]
-then
+check-run-command-as-root() {
   case "$HOMEBREW_COMMAND" in
-    analytics|install|reinstall|postinstall|link|pin|update|upgrade|vendor-install|create|migrate|tap|tap-pin|switch)
+    analytics|install|reinstall|postinstall|link|pin|update|upgrade|\
+    vendor-install|create|migrate|tap|tap-pin|switch)
```

Comment:
Would you mind sorting the entries alphabetically like I did in my code snippet? The order currently makes no sense (to me) and this is making it harder to see whether a command is included in this list or not.